client/src/components/v3/status/WirelessModuleStatusCol.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Col, Card, Accordion, Button, Table } from 'react-bootstrap';
+import OnlineStatusPill, { isOnlineT } from 'components/common/OnlineStatusPill';
+
+export interface WirelessModuleStatusProps {
+    moduleName: string,
+    isOnline: isOnlineT,
```

Comment:
Why `boolean | null`? I think it would be reasonable to require true or false here

Reply:
I assume the `null` indicates when the sensor is not connected. Thus it could be online or offline, but we don't know.